elado/vonyarcmarina/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: elado
+title: Eladó egy fejlesztés alatt álló új, balatoni kikötő
```

Comment:
Az URL-ben benne lesz, hogy `vonyarcmarina`. Akkor már a szövegbe is beírhatod, hogy Vonyarcon van. Vagy ha titok, akkor ne tedd az oldal címébe.